tests/virt/upgrade_custom/vgpu/conftest.py

```
@@ -74,3 +85,78 @@ def rhel_vm_for_upgrade_session_scope(
+):
+    yield from label_nodes(nodes=nodes_with_supported_gpus, labels={GPU_WORKLOAD_CONFIG_LABEL: "vm-vgpu"})
+
+
+@pytest.fixture(scope="session")
+def nvidia_vgpu_manager_ready_session_scope(nvidia_vgpu_manager_ds, nodes_with_supported_gpus):
+    """Wait until nvidia-vgpu-manager DaemonSet is ready on all vGPU nodes."""
+    wait_for_nvidia_vgpu_manager(
+        nvidia_vgpu_manager_ds=nvidia_vgpu_manager_ds,
+        nodes_with_supported_gpus=nodes_with_supported_gpus,
+    )
+    yield nvidia_vgpu_manager_ds
```

Ruff (0.15.17)

[warning] No teardown in fixture `nvidia_vgpu_manager_ready_session_scope`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)